Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -1054,14 +1054,28 @@ extension Purchases: @unchecked Sendable {}
 
 // MARK: Internal
 
+extension Purchases: InternalPurchasesType {
+
+    @available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.2, *)
+    internal func healthRequest() async throws {
+        do {
+            try await self.backend.healthRequest()
+        } catch {
+            throw NewErrorUtils.purchasesError(withUntypedError: error)
+        }
+    }
+
+}
+
+/// Necessary because `ErrorUtils` inside of `Purchases` finds the obsoleted type.
+private typealias NewErrorUtils = ErrorUtils
+
 internal extension Purchases {
 
     var isStoreKit1Configured: Bool {
         return self.paymentQueueWrapper.sk1Wrapper != nil
     }
 
-    #if DEBUG
```

Comment:
Just checking about this change

Reply:
Oh good catch, I think this was an old change I didn't need anymore, I can add it again.